packages/realm/bindgen/CMakeLists.txt

```diff
@@ -78,6 +78,7 @@ if(DEFINED CMAKE_JS_VERSION)
     include(NodeJSTargets)
 endif()
 
+# TODO is this option dead? Do we want to support compiling with it set to OFF?
```

Comment:
We never ended up implementing building from Core prebuilds so we might as well kill it.

Reply:
I'd like to keep that option open as a potential optimization when we'd eventually start exploring moving back to deferring compilation to the developer's machine, instead of shipping a prebuilt binary for RN (to remove our strong dependency on the jsi.h)

Reply:
I think this setup is wrong either way. This (used to) let you import a binary realm-core lib, but then still build the realm-js C++ code. I think we need to defer building both, rather than just realm-core. So I propose removing this option for now*, to make room for reintroducing deferred building correctly later

*Not _right now_ though. I'd rather land this PR, and have separate work to clean up the CML.txt file now that we no longer need to be concerned with merging changes with the non-bindgen version.

Reply:
I'm fine with that 👍 Prebuilding Core with a particular NDK does introduce a dependency on the stdlib ABI as we've noticed before, so I'm fine with doing this correctly later.